src/server/index.spec.ts

```diff
@@ -77,6 +77,23 @@ describe('server', () => {
       expect(response.statusCode).toBe(400)
     })
 
+    it('proper handles valid SQL that generates an error', async () => {
```

Comment:
"properly handles"? 😜

Reply:
lol, copy/pasted the broken english. Will fix them all